daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-discover-services.md

```diff
@@ -173,7 +153,69 @@ To invoke an application using Dapr, you can use the `invoke` API on any Dapr in
 
 Below are code examples that leverage Dapr SDKs for service invocation.
 
-{{< tabs Dotnet Java Python Go Javascript>}}
+{{< tabs Python JavaScript ".NET" Java  Go >}}
+
+{{% codetab %}}
+
+```python
+#dependencies
+import random
+from time import sleep
+import logging
+import requests
+
+#code
+logging.basicConfig(level = logging.INFO) 
+while True:
+    sleep(random.randrange(50, 5000) / 1000)
+    orderId = random.randint(1, 1000)
+        #Invoke a service
+        result = requests.post(
+           url='%s/orders' % (base_url),
```

Comment:
In all the examples you are now showing how the /orders method on the order processing service is called. If that is the change then
1)The diagram at the start needs to change to be more like the Quickstart. In other words Checkout calls order processing, not the other way around currently in this article. 
2) This Javascript example still shows checkout and not orders on the order-processing service
3) Check any changes in the other code